### ISSUE TITLE
Make this repo standalone instead of a fork

### ISSUE THREAD

**Issue by Rishabhg71** [openzim/javascript-libzim#71]
I am proposing this for easy pull request (PR) management in this repository. Every time I attempt to create a PR, it is raised against https://github.com/dattaz/libzim_wasm. Also it might be good for overall repo management

**Comment by Jaifroid** [comment]
@Rishabhg71 I'd be happy for this to be done, so long as we fully credit dattaz for his work. I agree that currently the creation of PRs is difficult, and making the repo standalone would solve this. @kelson42, would you agree to such a change?

Do you know how to make this change? I haven't looked into it myself.

**Comment by kelson42** [comment]
Agree, but I don't know how to easily disconnect from the original repository without recreating from scratch.

**Comment by Jaifroid** [comment]
I wonder if this is still current:

https://stackoverflow.com/a/68795659/9727685

(suggests a semi-automatic way of doing it via GitHub support bots).

**Comment by kelson42** [comment]
Done

**Comment by Jaifroid** [comment]
Oh wow, that's faster than I thought it would be! Thank you very much! 🙏🙏🙏
